tests/test_other.py

```diff
@@ -6686,6 +6686,40 @@ def test_emscripten_print_double(self):
 |-inf : 9 : -Infinity : -inf : 4|
 ''', out)
 
+  def test_emscripten_scan_stack(self):
```

Comment:
What does this test check that the test_core one doesn't?

Reply:
core has a check for registers, which is crucial as different opt levels may interact there. other has a test for the stack, which is just a silly trivial function that returns the C stack location, so it would be wasteful to run it in core (which would run it on all modes).